jakartaee11/JakartaEE11ReleasePlan.md

```diff
@@ -198,61 +200,60 @@ As stated earlier, all component Specifications which plan a Major or Minor rele
 
 ### Waves
 
-We are proposing to deliver Jakarta EE 11 in a set of waves similar to those delivered in the previous Jakarta EE releases. These waves are somewhat related to the dependency tree of specifications. We aim to deliver specifications with a low number of dependencies first followed by other specifications.
+We are proposing to deliver Jakarta EE 11 in a set of waves similar to those delivered in the previous Jakarta EE releases. These waves are somewhat related to the dependency tree of specifications. We aim to deliver specifications with a low number of dependencies first followed by other specifications. (updated specifications marked with asterisks)
 
 #### Wave 1
 
-- Jakarta Annotations
+- Jakarta Annotations*
 
 #### Wave 2
 
 - Jakarta Expression Language
```

Comment:
https://jakarta.ee/specifications/expression-language/6.0/ is new for EE 11 so Expression Language should have an asterik I think.